pybind11/setup_helpers.py

```diff
@@ -411,6 +411,9 @@ def _single_compile(obj):
 
             try:
                 import multiprocessing
+                import multiprocessing.synchronize  # Check for platforms like that have some
+                                                    # multiprocess capabilities but lack semaphores,
+                                                    # such as AWS Lambda and Android Termux.
```

Comment:
```suggestion
                # Check for platforms like that have some multiprocess capabilities but
                #  lack semaphores, such as AWS Lambda and Android Termux.
                import multiprocessing.synchronize
```

I don't think we need to import multiprocessing if we import multiprocessing.X. The comment is more readable on it's own line.

Reply:
Changed